sapi/fpm/fpm/fpm_unix.c

```diff
@@ -392,6 +402,105 @@ static int fpm_unix_conf_wp(struct fpm_worker_pool_s *wp) /* {{{ */
 }
 /* }}} */
 
+#if HAVE_FPM_CPUAFFINITY
+struct fpm_cpuaffinity_conf {
+#if defined(HAVE_SCHED_SETAFFINITY) || defined(HAVE_CPUSET_SETAFFINITY)
+	cpu_set_t cset;
+#endif
+	long min;
+	long max;
+};
+
+static long fpm_cpumax(void)
+{
+	static long cpuid = LONG_MIN;
+	if (cpuid == LONG_MIN) {
+		cpu_set_t cset;
+#if defined(HAVE_SCHED_SETAFFINITY)
+		if (sched_getaffinity(0, sizeof(cset), &cset) == 0) {
+#elif defined(HAVE_CPUSET_SETAFFINITY)
+		if (cpuset_getaffinity(CPU_LEVEL_WHICH, CPU_WHICH_PID, -1, sizeof(cset), &cset) == 0) {
+#endif
+			cpuid = CPU_COUNT(&cset);
+		} else {
+			cpuid = -1;
+		}
+	}
+
+	return cpuid;
+}
+
+static void fpm_cpuaffinity_init(struct fpm_cpuaffinity_conf *c)
+{
+#if defined(HAVE_FPM_CPUAFFINITY)
```

Comment:
What's the point of this actually. Isn't this already inside this block: https://github.com/php/php-src/pull/10075/files#diff-6b26b2bd2a798d905b4419198f73f61fac2ecc265b4d361d204f565fd91d407fR405

Reply:
Yeah so it needs to get removed...